### PR TITLE
Xaiki/0.5.1 packaging update

### PR DESCRIPTION
**Pull request by xaiki**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -1,7 +1,8 @@
 Source: libndn-cxx
 Priority: optional
 Maintainer: Endless Maintainers <maintainers@endlessm.com>
-Build-Depends: debhelper (>= 9),
+Build-Depends: adduser,
```
Comment (dbnicholson) [inline debian/control]: Why is adduser needed as a build dependency? AFAICT, the use of adduser is in ndnsec-tools.postinst, so this should be added to the ndnsec-tools `Depends`.

**Review by dbnicholson — CHANGES_REQUESTED** [review]
Just one comment. Also, remember to create and push a `Version_0.5.1` tag when you're ready or jenkins will just bomb because it's calculated version will be less than what's in debian/changelog.

File: debian/control (hunk 1 above)
Comment (xaiki) [reply debian/control]: you are absolutely right, silly mistake, fixed.

**Review by dbnicholson — APPROVED** [review]

**Comment by dbnicholson** [comment]
Looks good. Go ahead and make the tag and force push to master.

**Pull request merged**